### ISSUE TITLE
Aggregate count not work as expected with pagination

### ISSUE THREAD

**Issue by stearm** [prisma/prisma#3243]
**Describe the bug**
Aggregation `count` not working as expected: when you try to list an entity using generated query from Prisma with pagination (eg. `skip` + `first` parameters), count returns exactly the `first` parameter or lower number corresponding to the number of remaining items.

**To Reproduce**
```
query {
  listMyEntity (skip: 0, first: 20) {
    aggregate {
      count
    }
  }
}
```

count result is at most 20, even if I have 100000 elements.

**Expected behavior**
I would like to obtain the total number of items, as the previous versions of prisma worked.

**Versions (please complete the following information):**
 - OS: [`Ubuntu 16.04.4 LTS (GNU/Linux 4.4.0-1061-aws x86_64)`] 
 - `prisma` CLI: [`prisma/1.7.1 (darwin-x64) node-v10.4.0`]
 - `prisma`: 1.17.1, `prisma-binding`: 2.1.6, `graphql-yoga`: 1.16.2

**Comment by marktani** [comment]
This is the expected behaviour. The previous behaviour was not intended and recently fixed. You can query the full count and the filtered count in one request like so:

```graphql
query {
  listMyEntity (skip: 0, first: 20) {
    aggregate {
      count
    }
  }
  full: listMyEntity {
    aggregate {
      count
    }
  }
}
```

**Comment by prolink007** [comment]
@marktani What version did this change in? I am not seeing any reference in the documentation to a "full" on query. I am also using the latest version of prisma and not seeing "full" as an option in my playground.

Is this documentation still accurate for connections? Where is more details related to your claims? I am having trouble getting aggregate to work as expected and i am unable to find this "full" option on my connection queries.
https://www.prisma.io/docs/1.14/reference/prisma-api/queries-ahwee4zaey#connection-queries

**Comment by jhalborg** [comment]
@marktani - If the Prisma team is sticking with this decision, could you help out here? https://www.prisma.io/forum/t/how-to-get-a-full-aggregate-for-a-connection-using-prisma-bindings/5698 

TL;DR - How can you get the "real" aggregate count when using `prisma-binding`, passing the user's `info` object to your Prisma query?

**Comment by Fabiomad85** [comment]
I have the same problem here
